lib/Botkit.d.ts

```diff
@@ -221,13 +221,18 @@ declare namespace botkit {
     action?: string;
     channel?: string;
     match?: RegExpMatchArray;
+    replace_original?: boolean;
+    response_type?: string;
```

Comment:
These properties are Slack specific, please move them to SlackMessage interface.